tests/mocha/server/HelpersTest.js

```diff
@@ -0,0 +1,24 @@
+MochaWeb.testOnly(function(){
+  describe('helpers', function() {
+
+    describe('getCompetitionEvents', function() {
+      it('gets events', function() {
+        chai.expect(getCompetitionEvents(null)).to.deep.equal([]);
+
+        var compId = Competitions.insert({ competitionName: "Comp One", listed: false, startDate: new Date() });
+        chai.expect(getCompetitionEvents(compId)).to.deep.equal([]);
+
+        Rounds.insert({ competitionId: compId, eventCode: '333', formatCode: '2', roundCode: 'f', nthRound: 1 });
+        Rounds.insert({ competitionId: compId, title: 'non competition' });
+
+        chai.expect(getCompetitionEvents(compId)).to.deep.equal([{competitionId: compId, eventCode: '333'}]);
+
+        Rounds.insert({ competitionId: compId, eventCode: '222', formatCode: '2', roundCode: 'f', nthRound: 1 });
+
+        // This assumes Mongo returns things in this order. Order independent deep comparison in Chai is strangely hard.
```

Comment:
So in this case, `getCompetitionEvents` is defined to return events in a particular order (the ordering of events in [__wca.js](https://github.com/cubing/ccm/blob/master/both/__wca.js#L499)). So this is a good test to have!